src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -897,16 +909,33 @@ private static void validatePropertySectionPropertyJavadocTokens(String fileName
         assertEquals("javadoc tokens to check",
                 columns.get(1).getTextContent(), fileName + " section '" + sectionName
                         + "' should have the basic token javadoc description");
-        assertEquals("subset of javadoc tokens "
-                        + CheckUtil.getJavadocTokenText(check.getAcceptableJavadocTokens(),
-                                check.getRequiredJavadocTokens()), columns.get(2).getTextContent()
-                        .replaceAll("\\s+", " ").trim(), fileName + " section '" + sectionName
-                                + "' should have all the acceptable javadoc tokens");
-        assertEquals(
-                CheckUtil.getJavadocTokenText(check.getDefaultJavadocTokens(),
-                        check.getRequiredJavadocTokens()), columns.get(3).getTextContent()
-                        .replaceAll("\\s+", " ").trim(), fileName + " section '" + sectionName
-                                + "' should have all the default javadoc tokens");
+
+        String acceptableTokenText = columns.get(2).getTextContent().trim();
+        final int removePrefixIdx = "subset of javadoc tokens".length();
+        acceptableTokenText = acceptableTokenText.substring(removePrefixIdx);
+        assertFalse(isInvalidTokenPunctuation(acceptableTokenText),
+                ", & . should be placed at beginning of the next corresponding lines "
+                + "in " + fileName + " acceptable javadoc token section: " + sectionName);
+        assertEquals(CheckUtil.getJavadocTokenText(check.getAcceptableJavadocTokens(),
+                check.getRequiredJavadocTokens()).replaceAll("\\s+", ""),
+                acceptableTokenText.replaceAll("\\s+", ""),
+                fileName + " section '" + sectionName
+                        + "' should have all the acceptable javadoc tokens");
+
+        final String defaultTokenText = columns.get(3).getTextContent().trim();
+        assertFalse(isInvalidTokenPunctuation(defaultTokenText),
+                ", & . should be placed at beginning of the next corresponding lines "
+                + "in " + fileName + " default javadoc token section: " + sectionName);
```

Comment:
same x2.

Reply:
Done

Reply:
Not completely done, see above.

Reply:
Sorry, for missing out, I have resolved them.